config/Creatbot F430 V9.5.cfg

```diff
@@ -0,0 +1,254 @@
+# This file contains common pin mappings for the Creatbot F430
+# printer with V9.5 mainboard.
+
+# To use this config, the firmware should be compiled for the atmega2560.
+# When running "make menuconfig" you have to:
+#  Enable extra-low level options
+#  Micro-controller: Atmega AVR
+#  Processor model: atmega2560
+#  Processor speed: 16Mhz
+#  Communication: UART0
+#  Baud: 152000
+
+# See docs/Config_Reference.md for a description of parameters.
+
+#  NOTE: the following will no longer work
+#  Screen
+#  Udisk
+#  Power OFF switch (can use psu control in octoprint)
+
+[mcu]
+serial: /dev/ttyACM0
+baud: 115200
+
+[printer]
+kinematics: cartesian
+max_velocity: 200
+max_accel: 500
+max_z_velocity: 5
+max_z_accel: 100
+square_corner_velocity: 5.0
+
+## Steppers ##
+
+[stepper_x]
+step_pin: PH1
+dir_pin: !PH0
+enable_pin: !PL1
+microsteps: 16
+rotation_distance: 40.64
+endstop_pin: ^!PC0
+#endstop_pin: ^PE4

```

Comment:
Can you add a comment why this line is commented (different board config) or remove it if unneeded